app/src/main/res/drawable/continent_north_america.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:viewportWidth="845.6545"
+    android:viewportHeight="565.9996"
+    android:width="845.65448dp"
+    android:height="565.99963dp">

```

Comment:
These should be `64dp` if I remember correctly. Bonus points for using `64px` in inkscape already and centering the drawing in it.